part-1-ajax/test.js

```diff
@@ -1,76 +1,108 @@
-function getCountryName(code) {
+//LISA
+
+$('button').click(function(event) {
+    var input = $('input').val()
+    $('ul').empty();
+    $('input').val('');
+    $('span').empty();
+
+    console.log(input)
+    var country = input
     $.ajax({
-      //uses the three letter code to get the country's full name
-      }
+        url: 'https://www.restcountries.eu/rest/v1/name/' + country,
+        success: parseData
     })
-  }
 
-  function parseData(country) {
-    //takes the country data
-    //extracts the name, capital, population, and neighboring countries
-    //iterates through the neighbors array to send off second AJAX request
-  }
 
-  var button = document.getElementsByTagName('button')[0]
-  button.addEventListener('click', function(){
+    function parseData(country) {
+        var name = country[0].name;
+        var cap = country[0].capital;
+        var pop = country[0].population;
+        var neighbor = country[0].borders;
+        console.log(name, cap, pop, neighbor)
+        $('.name').append(name);
+        $('.cap').append(cap);
+        $('.pop').append(pop);
+        for (var i = 0; i < neighbor.length; i++) {
+            getCountryName(neighbor[i])
+        }
+        //takes the country data
+        //extracts the name, capital, population, and neighboring countries
+        //iterates through the neighbors array to send off second AJAX request
+        function getCountryName(code) {
+            $.ajax({
```

Comment:
good job lisa! you might want to separate the success callback functions from the AJAX calls. for example 

function callback() {}

$.ajax({
url: www.google.com,
success: callback
})

just so you have shorter functions